sdk/ml/azure-ai-ml/azure/ai/ml/operations/_feature_set_operations.py

```diff
@@ -159,6 +159,10 @@ def begin_create_or_update(self, featureset: FeatureSet, **kwargs: Dict) -> LROP
         featureset_copy.properties["featuresetProperties"] = json.dumps(featureset_spec._to_dict())
 
         sas_uri = None
+
+        with open(os.path.join(featureset_copy.path, ".amlignore"), mode="w", encoding="utf-8") as f:
```

Comment:
Does it override if there is already a file named ".amlignore"? We need it to be overriden.

Reply:
And is this folder the spec folder? I believe we only upload the spec folder. So the .amlignore needs to be under spec folder.

Reply:
yes, the mode is w, which means create or overwrite a file if exists.